src/meta/meta_service.cpp

```diff
@@ -257,6 +257,24 @@ void meta_service::get_node_state(/*out*/ std::map<rpc_address, bool> &all_nodes
 
 void meta_service::balancer_run() { _state->check_all_partitions(); }
 
+bool meta_service::try_lock_meta_op_status(ENUM_META_OP_STATUS op_status)
+{
+    ENUM_META_OP_STATUS expected = ENUM_META_OP_STATUS::FREE;
+    if (!_meta_op_status.compare_exchange_strong(expected, op_status)) {
+        ddebug_f("LOCK meta op status failed, meta server is busy, current op status is {}", (uint8_t)expected);
```

Comment:
I recommend using error log here, not debug log.

Reply:
Good idea